modules/pulsar/src/main/resources/proxy.conf

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
will it work without this file? If so, it would be better to avoid file mounting because in some rare environments that's not possible

Reply:
we do need it simply because we need to change the default port of the embedded proxy , otherwise we will have to manipulate the advertised address which I prefer less.

Reply:
Do you think it's possible to keep only changed properties? Currently it's a bit unclear what was changed compared to the default config :)

Reply:
I don't think so the best I can think of is to add comments

Reply:
I was able to do it like this:
```java
withEnv("servicePort", "6850");
withEnv("webServicePort", "8280");
withEnv("webServicePortTls", "8643");
withCommand("/bin/bash", "-c", "" +
    "bin/apply-config-from-env.py conf/proxy.conf && " +
    "bin/pulsar standalone & " +
    "bin/pulsar proxy --zookeeper-servers localhost:2181 --global-zookeeper-servers localhost:2181"
);

waitingFor(Wait.forLogMessage(".*Started Pulsar Proxy.*\\s", 1));
```

WDYT? :)

Reply:
that's fine will go with that.